lib/chargify_api_ares/response_helper.rb

```diff
@@ -1,5 +1,19 @@
 module Chargify
   module ResponseHelper
+    def save
```

Comment:
@jeremywrowe I'm wondering... could the new definition of `save` be done on `Base` rather than in `ResponseHelper`?  And is there a reason why `Subscription` (which doesn't include `ResponseHelper`) does the uniqueness token work on its own `save` method instead of using the one from `ResponseHelper`?